debian/rules

```diff
@@ -5,9 +5,24 @@ ifneq ($(shell echo ${CI}),true)
 	YELP_DH_INSTALL = install -d completions/usr debian/package/.
 endif
 
+# Use the dpkg version comparison algorithm (which is easily in reach) to compare the build
+# system version to the version from which python3-distutils is required (18.04 and higher),
+# and conditionally populate a substitution variable which is referenced in the control file.
+# On bionic, where Ubuntu's python3.7 is used, this is a virtual package for python3-distutils
+# (in bionic-updates). On jammy, this is a real package providing distutils for python3.8 from deadsnakes.
```

Comment:
`python3.8`? Isn't it 3.7 instead?

Reply:
err, i copy-paasta'd this from an internal repo and missed a py38->py37 reference :p